src/libraries/Native/AnyOS/Common/compiler.h

```diff
@@ -20,5 +20,17 @@
 #endif
 
 #ifndef PALEXPORT
+#ifdef TARGET_UNIX
 #define PALEXPORT __attribute__ ((__visibility__ ("default")))
-#endif // ifndef PALEXPORT
+#else
+#define PALEXPORT
+#endif
+#endif // PALEXPORT
+
+#ifndef EXTERN_C
```

Comment:
Nothing in libraries should be compiled as C++, so this seems unnecessary.

Reply:
When we compile from coreclr in Windows it is compiled as C++, that's why I had to add it.

Reply:
Ah, ok, I've missed the fact that the static library itself is build in coreclr too.